docs/members-example/MembersExample.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { ReactVaporStore } from '../ReactVaporStore';
+import { MembersCompositeActions } from './actions/MembersActions';
+import { MembersCompositeView } from './views/MembersCompositeView';
+
+export class MembersExample extends React.Component<any, any> {
+
+  constructor(props?: any, context?: any) {
```

Comment:
Instead of using the constructor and having to call super, we should use componentWillMount or componentDidMount

https://facebook.github.io/react/docs/react-component.html#the-component-lifecycle